### DISCUSSION TITLE
Weird flashing at the bottom bar?

### DISCUSSION THREAD

**Discussion by Hex4C** [microsoft/vscode-cpptools#12079] [Q&A] [answered]
Hey! Just wanna check if anyone else have had this issue when editing a c file. I've checked with some friends and they seem to have a similar issue and it seems to've become an issue from version 1.19.4 and onward, if I downgrade to 1.18.5 it resolves the issue. Is this an intended feature or did I miss to turn some setting on? It's quite hard to show since it's flashing for a split second and moves the bottom bar a little bit before returning to normal. Happens when you type and stop for a split second or save the file.

**Comment by bobbrow** [comment]
Unless it's the language bar showing that we're doing work for IntelliSense, I'm not sure that we know what you're seeing.  Is it possible to do a screen recording and share what you see?

**Reply by Hex4C** [reply]
Yes, probably should've sent in the first place, sorry about that. As mentioned if I go back to 1.18.5 the issue completely disappears.

https://github.com/microsoft/vscode-cpptools/assets/58385827/0157dcd7-9927-4658-b28d-ad1f889eb019

**Reply by bobbrow** [reply, marked as answer]
Thanks. I was able to walk through the frames and find out what it was.  It is the language bar showing that we're updating IntelliSense, but it happens so fast that it's quite distracting.  If VS Code doesn't smooth this out for us, we're probably going to need to add a delay timer so that it doesn't have this strobing effect.

![image](https://github.com/microsoft/vscode-cpptools/assets/12818240/7165cb37-2f5b-4faf-b9cc-f7b66c1372bc)

**Reply by Hex4C** [reply]
I see, well for now I'll roll with a previous version or hide the bar entirely whilst I'm working in C. Thanks for the quick reply!

**Reply by bobbrow** [reply]
I created an issue for it.  https://github.com/microsoft/vscode-cpptools/issues/12084
I consider this to be a relatively high priority.  We'll get a fix out as soon as we can.  Thank you for reporting it!

**Reply by bobbrow** [reply]
When this happens, what is the state of the editor and what are you doing at that time?  e.g. Do you have a small file and are typing one character at a time, or is this a large file?

**Reply by Hex4C** [reply]
I have one small C file opened and happens when I'm either typing or saving the file. In the clip I was spamming ctrl + s.

**Reply by bobbrow** [reply]
Thanks.  It sounds like we just need to add a minimum duration for the status change so it doesn't flicker like that.

**Reply by bobbrow** [reply]
This is fixed in 1.19.7

**Reply by Hex4C** [reply]
Awesome! Huge thanks for the quick reply and fix, will try it out as soon as it's released!

**Reply by bobbrow** [reply]
1.19.7 was released yesterday afternoon

**Reply by Hex4C** [reply]
Gave it a try and works like a charm! Huge thanks for the quick fix, it was really bothering me before.

**Reply by bobbrow** [reply]
Thank you for confirming.  Enjoy the updates in 1.19!
